content/administration/neutralized_database.rst

```diff
@@ -29,6 +29,8 @@ Here is a non-exhaustive list of the deactivated parameters:
 - payment providers
 - delivery methods
 - :abbr:`IAP (In-App Purchase)` tokens
+- search engine indexing (updating the robots.txt file to disallow all crawling and add a robots
+  meta tag with "noindex, nofollow")
```

Comment:
We probably don't need to go into technical details, and we can probably keep it shorter as for other points?
```suggestion
- website visibility (prevent search engines from indexing your site)
```

Just my2cents, it's purely about wording so take it as just my opinion ;-)

Reply:
@rdeodoo For sure! Putting myself in the customer's shoes makes sense.

Reply:
I'll let the doc review team decide what to do with the "parameters" wording.

Reply:
@SergeBayet @rdeodoo 
Indeed, "features" would be better :)

Reply:
@xpl-odoo It's corrected. Thanks